.github/workflows/pr_body_check.yml

```diff
@@ -7,6 +7,7 @@ on:
     types: [opened, edited, reopened, synchronize]
 jobs:
   pr-check:
+    if: github.repository_owner == 'ansible' && github.repository == 'awx'
```

Comment:
This was mistaken - the repository is "ansible/awx", so need to figure out how to deal with that.